docs-md/tutorials/basics-docker.md

```diff
@@ -522,18 +741,27 @@ docker run --network tutorials_default --rm  \
   confluentinc/ksql-examples:{{ site.release }} \
   ksql-datagen \
       quickstart=orders \
-      format=avro \
+      format=json \
```

Comment:
had to switch to Json as the key type is not a STRING, so schema inference does not work yet :(